src/network/clientpackethandler.cpp

```diff
@@ -1043,16 +1040,9 @@ void Client::handleCommand_DeleteParticleSpawner(NetworkPacket* pkt)
 	u32 server_id;
 	*pkt >> server_id;
 
-	u32 client_id;
-	auto i = m_particles_server_to_client.find(server_id);
-	if (i != m_particles_server_to_client.end())
-		client_id = i->second;
-	else
-		return;
```

Comment:
there should be a sanity check here for the unlikely case that the server id is > `m_next_particle_spawner_id`

Reply:
this will never happen in my PR because m_next_particle_spawner_id starts at U32_MAX + 1